python/social_bot/tasks.py

```diff
@@ -43,6 +103,7 @@ class GoalTask(teacher.Task):
     """
 
     def __init__(self,
+                 env,
                  max_steps=500,
```

Comment:
In the future, maybe we could put max_steps into Task, and put the main while loop in run into Task.run, and have a more modularized structure for different tasks?  Maybe define a task_success() function, etc..  For this change, no need to do that.

Reply:
thanks for the suggestin. Should have a unified modular structure for tasks.